### PR TITLE
Docs 7726 update mdip onboarding details pa

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

File: modules/ROOT/pages/configure-multiple-credential-providers.adoc [hunk 1]
```diff
@@ -28,16 +26,32 @@ If no external IdPs are enabled, Access Manager acts as the default authorizatio
 
 == The Use Case
 
-To better understand how to implement multiple external credential providers, imagine a company, "Good Weather" that uses meteorology to forecast changing weather. Good Weather collects quantitative data about the current state of the atmosphere at a given place and, using meteorology, predicts how that current state will change over time.
+To better understand how to implement multiple external client providers, imagine a company, "Good Weather" that uses meteorology to forecast changing weather. Good Weather collects quantitative data about the current state of the atmosphere at a given place and, using meteorology, predicts how that current state will change over time.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-multiple-credential-providers.adoc]: Good Weather should be fully parenthetical (two commas) or not (no commas):
```suggestion
To better understand how to implement multiple external client providers, imagine a company named "Good Weather" that uses meteorology to forecast changing weather. Good Weather collects quantitative data about the current state of the atmosphere at a given place and, using meteorology, predicts how that current state will change over time.
```

File: modules/ROOT/pages/configure-multiple-credential-providers.adoc [hunk 2]
```diff
@@ -28,16 +26,32 @@ If no external IdPs are enabled, Access Manager acts as the default authorizatio
...
 
 Because Good Weather must ensure that its weather-forecasting data is accurate and secure, the weather APIs are accessed only by employees of the Forecasting department. Good Weather also wants to ensure that the production APIs are the single source of truth at all times.
 
 To ensure security and access segregation for different organizations, environments, and APIs, Good Weather IT sets up separate external client providers for the Forecasting department.
 
-//example of how they implement the client providers.
-//Tomas how will they best implement this?
+== Onboarding Multiple Client Providers
+
+Your onbaording multiple client providers procedure differs based on how you have configured client providers and your business organizations in your company:
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-multiple-credential-providers.adoc]:
```suggestion
The procedure that you use to onboard multiple client providers differs based on how you have configured client providers and the business organizations in your company:
```

File: modules/ROOT/pages/configure-multiple-credential-providers.adoc [hunk 3]
```diff
@@ -28,16 +26,32 @@ If no external IdPs are enabled, Access Manager acts as the default authorizatio
...
+
+* *Scenario 1: No external providers configured:*
++
+Creating and modifying such environments and APIs behave the same. The native Anypoint Platform client provider is assigned for such environments. The existing APIs and environment have no impact.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-multiple-credential-providers.adoc]: To what does "behave the same" refer: creating and modifying? Environments and APIs? something else?
Please reword to clarify: Both creating and modifying involve the same steps? the results are the same? Or creating and modifying is the same for environments and APIs?

Comment (kbostrom) [reply modules/ROOT/pages/configure-multiple-credential-providers.adoc]: Or do you intend to say that creating and modifying environments and APIs is not affected by / has no effect on existing ones (which cannot be true in the case of modifying...).

File: modules/ROOT/pages/configure-multiple-credential-providers.adoc [hunk 4]
```diff
@@ -28,16 +26,32 @@ If no external IdPs are enabled, Access Manager acts as the default authorizatio
...
 
 To ensure security and access segregation for different organizations, environments, and APIs, Good Weather IT sets up separate external client providers for the Forecasting department.
 
-//example of how they implement the client providers.
-//Tomas how will they best implement this?
+== Onboarding Multiple Client Providers
+
+Your onbaording multiple client providers procedure differs based on how you have configured client providers and your business organizations in your company:
+
+* *Scenario 1: No external providers configured:*
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-multiple-credential-providers.adoc]: This seems to be an inline heading (bold) that also follows bullet list item style (sentence).

We really do not have a sanctioned format or process for inline headings, so I suggest that you make these bullet-list items, without the boldface and without the colon:

```suggestion
* No external client providers configured
```

File: modules/ROOT/pages/configure-multiple-credential-providers.adoc [hunk 5]
```diff
@@ -28,16 +26,32 @@ If no external IdPs are enabled, Access Manager acts as the default authorizatio
...
++
+Creating and modifying such environments and APIs behave the same. The native Anypoint Platform client provider is assigned for such environments. The existing APIs and environment have no impact.
+* *Scenario 2: External client provider configured:*
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-multiple-credential-providers.adoc]: ditto
```suggestion
* External client providers configured
```

File: modules/ROOT/pages/configure-multiple-credential-providers.adoc [hunk 6]
```diff
@@ -28,16 +26,32 @@ If no external IdPs are enabled, Access Manager acts as the default authorizatio
...
++ 
+If you have external client providers, such as PingFederate, configured for your environments, the client provider is automatically ported when you onboard the multiple client providers feature. However, when you create a new environment, you must manually assign the correct client provider to your environment in Access Management.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-multiple-credential-providers.adoc]:
```suggestion
If you have external client providers such as PingFederate configured for your environments, the client provider is automatically ported when you onboard the multiple client providers feature. However, when you create a new environment, you must manually assign the correct client provider to your environment in Access Management. 
```
HOWEVER...are you onboarding the FEATURE or the actual client providers?

File: modules/ROOT/pages/configure-multiple-credential-providers.adoc [hunk 7]
```diff
@@ -28,16 +26,32 @@ If no external IdPs are enabled, Access Manager acts as the default authorizatio
...
+Your onbaording multiple client providers procedure differs based on how you have configured client providers and your business organizations in your company:
+
+* *Scenario 1: No external providers configured:*
++
+Creating and modifying such environments and APIs behave the same. The native Anypoint Platform client provider is assigned for such environments. The existing APIs and environment have no impact.
+* *Scenario 2: External client provider configured:*
++ 
+If you have external client providers, such as PingFederate, configured for your environments, the client provider is automatically ported when you onboard the multiple client providers feature. However, when you create a new environment, you must manually assign the correct client provider to your environment in Access Management. 
+
+*Best Practice*
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-multiple-credential-providers.adoc:48]: Does this apply to one or both of the two environments listed? Clarify, but don't make this text with a heading, but just go straight into a new para.

File: modules/ROOT/pages/configure-multiple-credential-providers.adoc [hunk 8]
```diff
@@ -28,16 +26,32 @@ If no external IdPs are enabled, Access Manager acts as the default authorizatio
...
++
+To secure your APIs by leveraging the multiple client providers feature, create one external client provider per environment. Assign the existing client provider to your production environment, while adding the new providers to other environments, such as sandbox, and QA.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-multiple-credential-providers.adoc]:
```suggestion
To secure your APIs, the best practice is to create one external client provider per environment. Assign the existing client provider to your production environment when you add new providers to other environments, such as the sandbox, or when you perform QA. 
```
I assumed that "and QA" meant this, but perhaps you mean the QA environment? Clarify, if so.

File: modules/ROOT/pages/configure-multiple-credential-providers.adoc [hunk 9]
```diff
@@ -28,16 +26,32 @@ If no external IdPs are enabled, Access Manager acts as the default authorizatio
...
+Creating and modifying such environments and APIs behave the same. The native Anypoint Platform client provider is assigned for such environments. The existing APIs and environment have no impact.
+* *Scenario 2: External client provider configured:*
++ 
+If you have external client providers, such as PingFederate, configured for your environments, the client provider is automatically ported when you onboard the multiple client providers feature. However, when you create a new environment, you must manually assign the correct client provider to your environment in Access Management. 
+
+*Best Practice*
++
+To secure your APIs by leveraging the multiple client providers feature, create one external client provider per environment. Assign the existing client provider to your production environment, while adding the new providers to other environments, such as sandbox, and QA. 
+
+However, before changing any existing client providers, ensure that you check the impact on managed APIs and their consumers.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-multiple-credential-providers.adoc]: Will this cause any confusion with the opening text about new things not affecting existing things?

**Review by kbostrom — APPROVED** [review]
Done

File: modules/ROOT/pages/configure-multiple-credential-providers.adoc (hunk 9 above)
Comment (tbruno) [reply modules/ROOT/pages/configure-multiple-credential-providers.adoc]: Maybe we need to reference them to the guidelines section.

**Review by tbruno — APPROVED** [review]
LGTM 👍

File: modules/ROOT/pages/configure-multiple-credential-providers.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/configure-multiple-credential-providers.adoc]: Done! Thanks!

File: modules/ROOT/pages/configure-multiple-credential-providers.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/configure-multiple-credential-providers.adoc]: Done! Thanks!

File: modules/ROOT/pages/configure-multiple-credential-providers.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/pages/configure-multiple-credential-providers.adoc]: Done! Thanks!

File: modules/ROOT/pages/configure-multiple-credential-providers.adoc (hunk 4 above)
Comment (priyaayyar) [reply modules/ROOT/pages/configure-multiple-credential-providers.adoc]: Done! Thanks!

File: modules/ROOT/pages/configure-multiple-credential-providers.adoc (hunk 5 above)
Comment (priyaayyar) [reply modules/ROOT/pages/configure-multiple-credential-providers.adoc]: Done! Thanks!

File: modules/ROOT/pages/configure-multiple-credential-providers.adoc (hunk 6 above)
Comment (priyaayyar) [reply modules/ROOT/pages/configure-multiple-credential-providers.adoc]: No the feature. Thanks! Done!

File: modules/ROOT/pages/configure-multiple-credential-providers.adoc (hunk 7 above)
Comment (priyaayyar) [reply modules/ROOT/pages/configure-multiple-credential-providers.adoc:48]: Environment here means something totally different. This is a best practice of how to implement MiDP period. Doesn't matter what the user's current setup is.

File: modules/ROOT/pages/configure-multiple-credential-providers.adoc (hunk 8 above)
Comment (priyaayyar) [reply modules/ROOT/pages/configure-multiple-credential-providers.adoc]: You can secure your APIs in many different ways, such as by applying policies and such. Just mentioning "To secure your APIs" makes one feel that is the only way to secure your API. And plus the title of the section is already Best Practice. QA is also an environment like production.

File: modules/ROOT/pages/configure-multiple-credential-providers.adoc (hunk 9 above)
Comment (priyaayyar) [reply modules/ROOT/pages/configure-multiple-credential-providers.adoc]: No, I think that part is clear. Thanks! I'll add a reference, Tomas!

**Pull request merged**